lottie/src/main/java/com/airbnb/lottie/Content.java

```diff
@@ -3,5 +3,7 @@
 import java.util.List;
 
 interface Content {
+  String getName();
```

Comment:
It can, just wanted to make sure I understood your intention there since it doesn't seem like any PathContent names are used